src/use-case/my-script.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+# BASH3 Boilerplate
+#
+# This file:
+#
+#  - Is a template to write better bash scripts
+#  - Is delete-key friendly, in case you don't need e.g. command line option parsing
+#
+# More info:
+#
+#  - https://github.com/kvz/bash3boilerplate
+#  - http://kvz.io/blog/2013/02/26/introducing-bash3boilerplate/
+#
+# Version: 2.0.0
+#
+# Authors:
+#
+#  - Kevin van Zonneveld (http://kvz.io)
+#  - Izaak Beekman (https://izaakbeekman.com/)
+#  - Alexander Rathai (Alexander.Rathai@gmail.com)
+#  - Dr. Damian Rouson (http://www.sourceryinstitute.org/) (documentation)
+#
+# Usage:
+#
+#  LOG_LEVEL=7 ./main.sh -f /tmp/x -d
+#
+# Licensed under MIT
+# Copyright (c) 2013 Kevin van Zonneveld (http://kvz.io)
+
+# Incorporate the bash3boilerplate content that does not need to change from one use to the next
+# and designate the name of the file containing the current file's usage as the file with "-usage"
+# appended to the current file's name:
+# name followed by "-usage"
+__filename="$(basename "${BASH_SOURCE[0]}")"
+. bootstrap.sh -f ${__filename}-usage
+
+# Command-Line Interface (CLI) 
+# ----------------------------
+# This 'while' block below reads the file named in usage_page, which will also be used to 
+# parse CLI options, their associated arguments, their default values, and their status as
+# required or optional.  See the above-referenced bootstrap script for a comprehensive 
+# example of usage page contents.  See my-script.sh-usage for the specific usage page for
+# the current script.
+while IFS='' read -r line || [[ -n "$line" ]]; do
+  usage="${usage:-}""${line}"$'\n'
```

Comment:
What are the intentions here? I haven't tested this yet, but I think you'll be pulling in the `usage` variable previously defined by bootstrap.sh and then appending lines from the  my-srcript-usage file. I need to test this to confirm my suspicions though

Also, what does  the trailing `$'\n'` do?